src/commonControls/listControl/ListControl.ts

```diff
@@ -519,18 +519,19 @@ export class ListControl extends Control implements InteractionModelContributor
                         suggestions: ListFormatting.format(act.payload.choicesFromActivePage),
                     }),
             },
-            apl: {
-                enabled: true,
-                requestAPLDocument: generateTextListDocumentForListControl(),
+            apl: ListControlAPLPropsBuiltIns.APL_TextList((choiceId) => choiceId),
```

Comment:
Hmm agree, will rename it also I feel like TextList can just be const instead of a function and I will pass default slotIdMappers in namespace definition rather defining in ListControl.